10-inference.Rmd

```diff
@@ -438,7 +426,7 @@ grid.arrange(annotated_sampling_dist_20,
 )
 ```
 
-Based on the visualization, two points about the sample mean become clear.
```

Comment:
I think we need to use the term "sampling distribution" in this paragraph to emphasize to the students that what they're seeing is different sampling distributions, one for each size of sample